layouts/partials/fragments/hero.html

```diff
@@ -1,22 +1,22 @@
-{{ $data := ($.Scratch.Get "data") }}
-{{ "<!-- Hero -->" | safeHTML }}
-{{ $bg := $data.background | default "secondary"}}
+{{ $hero := .self.Params  }}
+{{ "<!-- hero -->" | safeHTML }}
+{{ $bg := $hero.background | default "secondary" }}
 
 <header>
-  {{ with $data.header }}
+  {{ with $hero.header }}
     <div class="jumbotron text-center header-image mb-0">
   {{ else }}
     <div class="jumbotron text-center mb-0">
   {{ end }}
-  {{ if $data.particles }}
+  {{ if $hero.particles }}
     <div id="particles-js"></div>
   {{ end }}
-  {{ with $data.logo }}
+  {{ with $hero.logo }}
     <img class="overlay img-fluid" src="{{ (printf "images/%s" .) | relURL }}"
-      {{- if $data.logoHeight -}} height="{{ $data.logoHeight }}"{{- end -}}
-      {{- if $data.logoWidth -}} width="{{ $data.logoWidth }}"{{- end -}}
+      {{- if $hero.logoHeight -}} height="{{ $hero.logoHeight }}"{{- end -}}
+      {{- if $hero.logoWidth -}} width="{{ $hero.logoWidth }}"{{- end -}}
```

Comment:
logoHeight and logoWidth need to be lowercased. There might be other instances similar to this so. Just stumbled upon this creating a test site.